test/services/return-requirements/purpose.service.test.js

```diff
@@ -0,0 +1,75 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+const Sinon = require('sinon')
+
+const { describe, it, beforeEach, afterEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Test helpers
+const DatabaseSupport = require('../../support/database.js')
+const SessionHelper = require('../../support/helpers/session.helper.js')
+
+// Things we need to stub
+const FetchPurposesService = require('../../../app/services/return-requirements/fetch-purposes.service.js')
+
+// Thing under test
+const SelectPurposeService = require('../../../app/services/return-requirements/purpose.service.js')
+
+describe('Select Purpose service', () => {
+  let session
+
+  beforeEach(async () => {
+    await DatabaseSupport.clean()
+    session = await SessionHelper.add({
+      data: {
+        licence: {
+          id: '8b7f78ba-f3ad-4cb6-a058-78abc4d1383d',
+          currentVersionStartDate: '2023-01-01T00:00:00.000Z',
+          endDate: null,
+          licenceRef: '01/ABC',
+          licenceHolder: 'Turbo Kid',
+          startDate: '2022-04-01T00:00:00.000Z',
+          licencePurposes: [
```

Comment:
The purpose.service calls two other services, the `FetchPurposesService` which is stubbed and `SelectPurposePresenter` which isn't stubbed. The presenter needs to be stubbed to test this file independently.